src/Germany/Mews.Fiscalizations.Germany/V2/FiskalyClient.cs

```diff
@@ -91,18 +96,18 @@ public Task<ResponseResult<Transaction>> StartTransactionAsync(AccessToken token
         {
             return Client.ProcessRequestAsync<Dto.TransactionRequest, Dto.TransactionResponse, Transaction>(
                 method: HttpMethod.Put,
-                endpoint: $"tss/{tssId}/tx/{transactionId}",
+                endpoint: $"tss/{tssId}/tx/{transactionId}?tx_revision=1",
                 request: RequestCreator.CreateTransaction(clientId),
                 successFunc: response => ModelMapper.MapTransaction(response),
                 token: token
             );
         }
 
-        public Task<ResponseResult<Transaction>> FinishTransactionAsync(AccessToken token, Guid clientId, Guid tssId, Bill bill, Guid transactionId, string lastRevision)
+        public Task<ResponseResult<Transaction>> FinishTransactionAsync(AccessToken token, Guid clientId, Guid tssId, Bill bill, Guid transactionId)
         {
             return Client.ProcessRequestAsync<Dto.FinishTransactionRequest, Dto.TransactionResponse, Transaction>(
                 method: HttpMethod.Put,
-                endpoint: $"tss/{tssId}/tx/{transactionId}?last_revision={lastRevision}",
+                endpoint: $"tss/{tssId}/tx/{transactionId}?tx_revision=2",
```

Comment:
This definitely shouldn't be hard-coded. The library should be universally usable and allow updates between transaction start and finish.

Reply:
That is right if we call the endpoint UpdateTransaction

But we call it FinishTransaction and actually everything is hard coded to finish the transaction.

Reply:
But that is pretty specific to our use-case, right? The library should be universally usable.

Reply:
Yes i completely agree, but changing this now won't help anyone, it just makes it confusing.

If we want to do it properly, we change rename this to UpdateTransactionAsync  and make it accept the transaction state and revision..etc but if we decide to do this, its definitely a separate PR.